app/controllers/menus_controller.rb

```diff
@@ -6,6 +6,10 @@ def new
     @menu = Menu.new
   end
 
+  def show
+    @menu = Menu.find(params[:id])
```

Comment:
Repeated in several methods. Plz extract to a private method and set as before action